x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -23,6 +24,7 @@ public EqlFunctionRegistry() {
         // Scalar functions
         // String
             new FunctionDefinition[] {
+                def(StringContains.class, StringContains::new, "stringcontains"),
```

Comment:
same convention as the existing? I believe function names are looked up case insensitive anyway
```suggestion
                def(StringContains.class, StringContains::new, "stringContains"),
```